core/src/main/scala/org/apache/spark/deploy/history/HistoryPage.scala

```diff
@@ -39,8 +39,6 @@ private[history] class HistoryPage(parent: HistoryServer) extends WebUIPage("")
     val allAppsSize = allApps.size
```

Comment:
We should be3 able to remove all the stuff above this right?  Its getting page parameter which should be done on datatables now and you don't need the list of application here.

Same with some stuff below, this. We don't need to calculate page coutn ,etc.